tests/login.go

```diff
@@ -179,21 +177,21 @@ func ReLoggedInFedoraExpecter(vmi *v1.VirtualMachineInstance, timeout int) (expe
 	return expecter, err
 }
 
-func configureConsole(expecter expect.Expecter, prompt string, shouldSudo bool) error {
+func configureConsole(expecter expect.Expecter, shouldSudo bool) error {
 	sudoString := ""
 	if shouldSudo {
 		sudoString = "sudo "
 	}
 	batch := append([]expect.Batcher{
 		&expect.BSnd{S: "stty cols 500 rows 500\n"},
-		&expect.BExp{R: prompt},
+		&expect.BExp{R: PromptExpression},
 		&expect.BSnd{S: "echo $?\n"},
 		&expect.BExp{R: RetValue("0")},
 		&expect.BSnd{S: fmt.Sprintf("%sdmesg -n 1\n", sudoString)},
-		&expect.BExp{R: prompt},
+		&expect.BExp{R: PromptExpression},
 		&expect.BSnd{S: "echo $?\n"},
 		&expect.BExp{R: RetValue("0")}})
-	resp, err := expecter.ExpectBatch(batch, 30*time.Second)
+	resp, err := ExpectBatchWithValidatedSend(expecter, batch, 30*time.Second)
```

Comment:
`configureConsole` shouldn't use `ExpectBatchWithValidatedSend`. `configureConsole` config things so that is will be possible to use `ExpectBatchWithValidatedSend`. We cannot use `ExpectBatchWithValidatedSend` before/during the configuration stage.

Reply:
If you can, I prefer the second commit (tests, expecter: add changes to safe expecter) will be removed from the PR.